functions/go/Makefile

```diff
@@ -61,6 +63,29 @@ push: $(FUNCTION_PUSH) ## Push images to registry. WARN: This operation should o
 $(FUNCTION_PUSH):
 	$(MAKE) CURRENT_FUNCTION=$(subst -PUSH,,$@) func-push
 
+.PHONY: check-licenses
+check-licenses: go-licenses $(FUNCTION_CHECKLICENSES)
+
+.PHONY: $(FUNCTION_CHECKLICENSES)
+$(FUNCTION_CHECKLICENSES):
+	$(MAKE) CURRENT_FUNCTION=$(subst -CHECKLICENSES,,$@) func-check-licenses
+
+# find or download go-licenses
+go-licenses:
```

Comment:
This seems like a hack. Can you just curl the pre-built binary? Note that unlike other tests, you don't need to run this across platforms/arch.

Reply:
> Can you just curl the pre-built binary?

No. Unfortunately, they don't release pre-built binary.

Using `go get` is their suggested way to install it: https://github.com/google/go-licenses#installation

Reply:
OK, could you extract lines 76 to 83 into a bash script? ./download_go_license.sh

In general mixing bash and Makefile makes things unreadable.